### DISCUSSION TITLE
休館日や年間予定の設定について

### DISCUSSION THREAD

**Discussion by Fumiko-Nishiura** [next-l/enju_leaf#1546] [Q&A] [answered]
お世話になっております。
休館日や年間予定表を設定する場合は、「催し物のインポート用ファイルの新規作成」にファイルを挿入する方法で合ってますでしょうか？　その場合、学校の年間予定表はExcelの表形式ですが、それをテキストファイルに変換するとよいのでしょうか？
カレンダー形式の方がよいのでしょうか？

もう一つ教えてください。
槐の赤いキャラクターは、「実」ですか？

**Comment by masao** [comment]
はい、その方法で設定いただくことになります。
カレンダー形式には対応していませんので、Excelのデータをさらにインポート用に修正いただくと良いと思います。
具体的には https://next-l.github.io/manual/1.3/enju_operation_7.html#section7_import_tsv にあるTSV形式のフォーマットにしていただくと良いと思います。

あとキャラクターのデザイナーさんからは、槐の実が広がっていくイメージを意識してデザインしたものと聞いた気がします。

**Comment by Fumiko-Nishiura** [comment, marked as answer]
高久先生

西浦です。お世話になっております。
ご回答をありがとうございました。
カレンダー形式にしなくてもよいことが分かって有難いです。
新年度開始のご多忙な時期に、お手数をおかけして申し訳ございません。

槐の実が広がっていくイメージ、分かりました。

次に、統計に関して一般図書館とは異なるところがあり、
後日質問させて頂きます。

取り合えず、1学期は旧システムと併用することに致しました。
引き続きよろしくお願い申し上げます。



2021年4月4日(日) 17:16 Masao Takaku ***@***.***>:

> はい、その方法で設定いただくことになります。
> カレンダー形式には対応していませんので、Excelのデータをさらにインポート用に修正いただくと良いと思います。
> 具体的には
> https://next-l.github.io/manual/1.3/enju_operation_7.html#section7_import_tsv
> にあるTSV形式のフォーマットにしていただくと良いと思います。
>
> あとキャラクターのデザイナーさんからは、槐の実が広がっていくイメージを意識してデザインしたものと聞いた気がします。
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/next-l/enju_leaf/discussions/1546#discussioncomment-567332>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ATNPS4X4R3NO73NEM5AAAC3THAN7NANCNFSM42DQQLCQ>
> .
>


-- 

─────────────────── ☆

西浦 富美子 / NISHIURA Fumiko

Email: ***@***.***

☆☆ ──────────────────

**Comment by Fumiko-Nishiura** [comment]
高久先生

青翔高校の図書館ボランティアの西浦と申します。
お世話になっております。お仕事中に失礼いたします。

先程青翔高校司書から連絡があり、
本日突然下記のようなメッセージが表示され、
困っているとのことでした。

このサーバーが seisho-hs.dla.jp
であることを確認できませんでした。セキュリティ証明書の期限が昨日で切れています。原因として、設定が不適切であるか、悪意のあるユーザーが接続を妨害していることが考えられます。パソコンの時計は現在
> 2021年4月20日火曜日 に設定されています。この時刻が正しくない場合は、システムの時計を修正して、このページを更新してください。
>
> seisho-hs.dla.jp にアクセスする（安全ではありません）
>
  セキュリティ証明書の期限 とは何でしょうか？
対処の方法を、ご教示いただけますようお願い申し上げます。


-- 

─────────────────── ☆

西浦 富美子 / NISHIURA Fumiko

Email: ***@***.***

☆☆ ──────────────────

**Reply by masao** [reply]
サーバ側のSSL証明書といって、https経由でパスワードなどを暗号化して送ることができるための仕組みがあるのですが、その有効期限が切れていたようです。先ほど田辺さんに更新してもらいましたので、いまは正常にアクセスできるのではないかと思います。

**Comment by Fumiko-Nishiura** [comment]
高久先生

西浦です。
早々にご対処頂き、ありがとうございました。
正常にアクセスでき、青翔高校とも連絡がつきました。
引続きよろしくお願い致します。
西浦

2021年4月20日(火) 16:46 Masao Takaku ***@***.***>:

>
> サーバ側のSSL証明書といって、https経由でパスワードなどを暗号化して送ることができるための仕組みがあるのですが、その有効期限が切れていたようです。先ほど田辺さんに更新してもらいましたので、いまは正常にアクセスできるのではないかと思います。
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/next-l/enju_leaf/discussions/1546#discussioncomment-633762>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ATNPS4VVGOIYI4POUPHGMSLTJUWM5ANCNFSM42DQQLCQ>
> .
>


-- 

─────────────────── ☆

西浦 富美子 / NISHIURA Fumiko

Email: ***@***.***

☆☆ ──────────────────
